flutter_local_notifications/ios/Classes/FlutterLocalNotificationsPlugin.m

```diff
@@ -96,6 +97,13 @@ @implementation FlutterLocalNotificationsPlugin {
 NSString *const PRESENTATION_OPTIONS_USER_DEFAULTS =
     @"flutter_local_notifications_presentation_options";
 
+NSString *const CHECK_ENABLED = @"isEnabled";
```

Comment:
thanks for making the change. minor thing but can the variables be renamed in the iOS and macOS implementation be renamed to be similar their values e.g. rename `CHECK_SOUND_ENABLED` to  `IS_SOUND_ENABLED`? the current naming reads like flags that the plugin should use check if the corresponding permission is enabled instead of a result

Reply:
Done